modules/apps/product-navigation/product-navigation-taglib/src/main/resources/META-INF/resources/control_menu/page.jsp

```diff
@@ -41,7 +41,7 @@ for (ProductNavigationControlMenuCategory productNavigationControlMenuCategory :
 %>
 
 <c:if test="<%= hasControlMenuEntries %>">
-	<div class="control-menu-container">
+	<div class="cadmin control-menu-container">
```

Comment:
Same as for JS components. @p2kmgcl provided a `layout:shadow` tag in his approach, so if we were to take this, something like `layout:isolate` or `layout:admin` or whatever declarative name we can come up with sounds better and more obvious to me than a simple css class.